thunderbird/processes/wps_generate_climos.py

```diff
@@ -185,6 +164,11 @@ def collect_args(self, request):
             loglevel,
         )
 
+    def dry_output_filename(self, filepath):
+        return os.path.join(
+            self.workdir, os.path.basename(filepath).split(".")[0] + "_dry.txt",
+        )
```

Comment:
This is awesome! We could probably benefit by moving it to `utils` and using it in other places as well! We could maybe even generalize it a bit and have the user be able to specify a name if they like. If not it would default to the behaviour you have right now.